service_mesh/v2x/ossm-reference-smcp.adoc

```diff
@@ -3,9 +3,29 @@
 include::modules/ossm-document-attributes.adoc[]
```

Comment:
I know you didn't change it in this PR, but maybe consider changing "SMCP" in the title to "Service Mesh control plane"?
It's not immediately obvious what this means IMP